lib/phoenix_live_view/utils.ex

```diff
@@ -42,7 +42,7 @@ defmodule Phoenix.LiveView.Utils do
   end
 
   defp configure_assigns(assigns, view, action) do
-    assigns |> Map.put(:live_view_module, view) |> Map.put(:live_view_action, action)
+    Map.merge(assigns, %{live_view_module: view, live_view_action: action, flash: %{}})
```

Comment:
Just FYI, Map.put is more efficient in Elixir, not that it would matter here. :)